src/darker/__main__.py

```diff
@@ -561,9 +561,18 @@ def main(  # noqa: C901,PLR0912,PLR0915
             msg = f"Path(s) {missing_reprs} do not exist in {rev2_repr}"
             raise FileNotFoundError(msg)
 
+    common_root_ = (
+        # On Windows, Python <= 3.9 requires the `filter_python_files` `root` argument
+        # to be an absolute path. Remove this after dropping support for Python 3.9.
+        # See https://github.com/python/cpython/issues/82852
+        common_root.resolve()
+        if WINDOWS and sys.version_info < (3, 10)
+        else common_root
+    )
```

Comment:
According to the issue you linked, this bug also affects Python 3.10

Reply:
For some reason the unit tests only failed on Python 3.8 and 3.9 in the Windows runners of the Github workflow. I'll look into this, thanks for the heads up!

Reply:
@DeinAlptraum I checked this; python/cpython#82852 was completed on August 7th, 2021, and Python 3.10 was released, including that bugfix, on October 4th, 2021.

So the bug was in the developing version 3.10 but was fixed before the release.